pis.yaml

```diff
@@ -0,0 +1,93 @@
+# Building
```

Comment:
We probably would not include your specific building config as part of the repo, and if we did we would put it in a samples directory. This should be removed for the actual PR.

Reply:
Thanks for the hint. I didn't intended to put this one in the pull request. However, I just removed it.